scene/2d/animated_sprite_2d.h

```diff
@@ -60,8 +60,6 @@ class AnimatedSprite2D : public Node2D {
 	void _calc_frame_speed_scale();
 	void _stop_internal(bool p_reset);
 
-	Rect2 _get_rect() const;
```

Comment:
Moving this method is not necessary to expose it. You can bind private methods too.